pyproject.toml

```diff
@@ -6,10 +6,10 @@ build-backend = "setuptools.build_meta"
 name = "neso_solar_consumer"
 version = "0.1"
 dependencies = [
-    "pandas",
+    "pandas"
```

Comment:
I think for the moment, it would be goo to have all requirements in here. This means we don't have to have requirements and dev-requirements files.